src/main/java/frc/robot/commands/vision/VisionAim.java

```diff
@@ -43,17 +43,18 @@ public VisionAim(Robot robot)
   public void initialize()
   {
    robot.vision.enableAimingLight();
+   robot.vision.resetPID();
   }
 
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute()
   {
-    robot.vision.calcAlign(robot.drivetrain.gyroAngle());
+    if (!robot.shooter.isAutoShooting()) robot.vision.calcAlign(robot.drivetrain.gyroAngle());
```

Comment:
Why only when not auto shooting?

Reply:
The auto shooting command is already running `calcAlign()`, and I don't want the method to run twice in one scheduler cycle, since it would effectively double the I term in the PID Controller.